cypress/e2e/home_page_spec.cy.ts

```diff
@@ -0,0 +1,5 @@
+describe('The Home Page', () => {
+    it('successfully loads', () => {
+        cy.visit('http://localhost:4200')
```

Comment:
```suggestion
        cy.visit('http://localhost:4200/home')
```
The page would redirect but shouldn't the explicit route be entered rather then just the base localhost domain?

Reply:
I'm fine with that.